CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+## Unreleased
+
+## v0.23.0
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.23.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.23.0) and the [opentelemetry-collector-contrib v0.23.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.23.0) releases.
+
+## 🛑 Breaking changes 🛑
+
+- Renamed default config from `splunk_config_linux.yaml` to `gateway_config.yaml` (#170)
+
+## 💡 Enhancements 💡
```

Comment:
Added agent bundle to deb and rpm's right?

Reply:
> Added agent bundle to deb and rpm's right?

Do you mean line 15?